### PR TITLE
adjusted readme to be csharp specific

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+`TestTaskTwo` verifies the work done to complete the List Views.  This includes displaying a list of jobs.
+
+`TestTaskThree` verifies work done to complete the `SearchController`.  This includes verifying how the `Results()` method was created and making sure that the method uses local variables. This test class focuses on checking your implementation.
+
+`TestTaskFour` verifies the behaviors required for `Results()` to pass accurate data to a view.
+ 
+### Appearance and Code Check:
+ 
+[Working Demo App](https://csharp-mvc.launchcodetechnicaltraining.org/)
```
Comment (codinglikeagirl42) [inline README.md]: https://csharp-mvc.launchcodetechnicaltraining.org/. I'm getting an error when I click on Search or List, is that suppose to happen?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ 
+1. Start the application.
+1. Test the search functionality of the project:
+
+    a. Initiate a search by location, using the search term "new york". Only 1 result should be returned on the page. The result may vary in appearance, but make sure it contains the job data organized similar to this image:
+
+    ![Search result](searchByLocation.png "Search Result Sample")
```
Comment (codinglikeagirl42) [inline README.md:32]: File seems to be missing.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# TechJobs, MVC Edition
+
+For [this assignment](https://education.launchcode.org/csharp/assignments/tech-jobs-mvc.html) students are asked to once again refactor a TechJobs application, this time using an MVC
```
Comment (codinglikeagirl42) [inline README.md]: Link doesn't work.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    
+    d. What strategy did you use to display the jobs. Did you create one table for all of the jobs displayed or one table per job?
+ 
+1. Back in the running app, navigate to the ``List`` view and select *View All*.
+
+    a. A view of all 98 jobs should be returned.
+
+## Submitting Your Work
+
+To submit your work please following the [Submission Instructions](https://education.launchcode.org/csharp/assignments/hello-world.html#submitting-your-work)
```
Comment (codinglikeagirl42) [inline README.md]: Link doesn't work.

**Review by codinglikeagirl42 — CHANGES_REQUESTED** [review]
This was a brief review of the readme with anticipation of changes in Q1. At which time, the readme will need to be resubmitted for a formal QA review.

File: README.md (hunk 1 above)
Comment (codinglikeagirl42) [reply README.md]: Please add verbiage about why students may run into issues with the app.

Comment (gildedgardenia) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (gildedgardenia) [reply README.md:32]: Fixed!

File: README.md (hunk 3 above)
Comment (gildedgardenia) [reply README.md]: Fixed!

File: README.md (hunk 4 above)
Comment (gildedgardenia) [reply README.md]: Fixed!

**Review by codinglikeagirl42 — APPROVED** [review]

**Pull request merged**